spec/presenters/idv/in_person/ready_to_verify_presenter_spec.rb

```diff
@@ -15,23 +16,33 @@
       profile: profile,
       enrollment_code: enrollment_code,
       unique_id: InPersonEnrollment.generate_unique_id,
-      created_at: created_at,
+      enrollment_established_at:  enrollment_established_at,
```

Comment:
I think you should have both `created_at` and `enrollment_established_at` defined here, because there will never be a record created without a `created_at` date.